src/tests/ftest/deployment/target_failure.py

```diff
@@ -248,6 +252,10 @@
         self.pool.reintegrate(rank="1", tgt_idx="0")
         self.pool.measure_rebuild_time(operation="Reintegrate 1 target", interval=5)
 
+        # Container status doesn't automatically go back to healthy after reintegrate. We
+        # need to manually set it.
+        self.container.set_prop(prop='status', value="healthy")
```

Comment:
(pylint-no-member) Instance of 'TestContainer' has no 'set_prop' member; maybe 'get_prop'?
(pylint-no-member) Instance of 'list' has no 'set_prop' member